<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pageCustomizePopular</class>
 <widget class="QWidget" name="customizePopularPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>380</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>380</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Popular</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ribbonsample.qrc">
    <normaloff>:/res/popularpage.png</normaloff>:/res/popularpage.png</iconset>
  </property>
  <property name="statusTip">
   <string>Change the most popular options</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="labelTitle">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Top options for Working with Demo&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="margin">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="labelTheme">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>30</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>&amp;Theme:</string>
        </property>
        <property name="buddy">
         <cstring>comboBoxTheme</cstring>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBoxTheme">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>28</y>
          <width>219</width>
          <height>23</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="labelAccentColor">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>63</y>
          <width>171</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Accen&amp;t Color (Office 2013 only):</string>
        </property>
        <property name="buddy">
         <cstring>pushButtonAccentColor</cstring>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonAccentColor">
        <property name="geometry">
         <rect>
          <x>353</x>
          <y>61</y>
          <width>17</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="labelBackgroundImage">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>98</y>
          <width>191</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>&amp;Background Image (Office 2013 only):</string>
        </property>
        <property name="buddy">
         <cstring>comboBoxBackground</cstring>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBoxBackground">
        <property name="geometry">
         <rect>
          <x>246</x>
          <y>96</y>
          <width>124</width>
          <height>23</height>
         </rect>
        </property>
       </widget>
       <widget class="QFrame" name="frameAccentColor">
        <property name="geometry">
         <rect>
          <x>246</x>
          <y>61</y>
          <width>105</width>
          <height>23</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ribbonsample.qrc"/>
 </resources>
 <connections/>
</ui>
